spec/1/controllers/main.rb

```diff
@@ -6,7 +6,7 @@ class MainController < Kenji::Controller
   end
 
   get '/crasher' do
-    raise
+    raise 'kaboom!'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.